evm_arithmetization/src/cpu/kernel/tests/core/jumpdest_analysis.rs

```diff
@@ -87,7 +87,7 @@ fn test_jumpdest_analysis() -> Result<()> {
         .get_mut(&CONTEXT)
         .unwrap()
         .pop();
-    interpreter.push(U256::one());
+    interpreter.push(41.into());
```

Comment:
Why is it changed?

Reply:
With the unification, the proofs are checked in the reverse order. I'm still not sure why, but it must come from a previous discrepancy between the interpreter and witness generation that we hadn't identified...

Reply:
cc @4l0n50 in case you have an explanation

Reply:
Hmmm. Maybe is because the jumpdests where sorted at some point but is not done anymore? (as it was no necessary)